### ISSUE TITLE
JSONPath branding proposal

### ISSUE THREAD

**Issue by graza** [ietf-wg-jsonpath/draft-ietf-jsonpath-base#227]
I was writing up documentation for my own implementation, and I wanted a favicon so that I could find the browser tab easily.  Rather than something random, I felt it should be something with at least some meaning.

Branding is important for a project and a big part of that is a recognisable logo.  Projects like JSON and JSONSchema have their own logos.  Why shouldn't JSONPath?  JSON and JSONShchema's logos:

![json](https://user-images.githubusercontent.com/147433/179801201-f2cd81fa-d477-44b7-8c10-d037c98d9516.png) ![jsonschema](https://user-images.githubusercontent.com/147433/179801353-bde99881-e7fa-4446-b87a-408e5799679b.png)

I put a poll to some creative friends and one of them came up with this:
![image](https://user-images.githubusercontent.com/147433/179801978-5a269dfc-646d-412d-88b1-a02eb5fba92c.png)

Below is what I came up with, heavily inspired by JSONSchema but also my friend's idea.  The slashes are suggestive of a filesystem path.  Obviously JSONPath doesn't use slashes (unlike XPath) but they look good in a favicon:

![JSONPath_logo2](https://user-images.githubusercontent.com/147433/179799907-e1d96a0a-1d47-422f-ac72-8444c3a9868d.png)

There were a bunch of others but none were as simple, neat, and tidy.  I find this one to be very easily recognisable.

Any other ideas out there?

**Comment by glyn** [comment]
I came up with this logo a while back:
![jsonpath-standard-icon](https://user-images.githubusercontent.com/94953/179809283-c07cf11e-087e-493a-8707-6f5c467c58be.png)

**Comment by graza** [comment]
Nice one @glyn.  Is that the accepted logo and I should close this?  What about without the arrow and two full stops instead of three?  Better for a favicon?

# {..}

**Comment by glyn** [comment]
No, it's not the accepted logo. It's what I knocked up in a few minutes when I created the jsonpath-standard github organisation.

I have no idea what makes a good favicon. I was quite keen on the arrow because of the path/reference connotations of JSONPath.

**Comment by cabo** [comment]
I must admit I really like the fact that {//} can be keyboarded.

**Comment by gregsdennis** [comment]
`{..}` can be keyboarded as well, and the double-dot is a thing in the syntax.

**Comment by glyn** [comment]
At the 2022-10-18 interim meeting it was agreed that we should close this issue until the draft is complete and then re-open it.
